### ISSUE TITLE
Can domain name expiration monitoring be supported

### ISSUE THREAD

**Issue by ivanmissu** [louislam/uptime-kuma#3628]
### ⚠️ Please verify that this bug has NOT been raised before.

- [X] I checked and didn't find similar issue

### 🛡️ Security Policy

- [X] I agree to have read this project [Security Policy](https://github.com/louislam/uptime-kuma/security/policy)

### 📝 Describe your problem

Can domain name expiration monitoring be supported ???

### 🐻 Uptime-Kuma Version

1.22.1

### 💻 Operating System and Arch

centos7.9 x86_64

### 🌐 Browser

Chrome 116.0.5845.96

### 🐋 Docker Version

docker 20

**Comment by CommanderStorm** [comment]
Please verify that this bug has not been raised before: 
https://github.com/louislam/uptime-kuma/issues?q=is%3Aissue+domain+name+expiration+

TLDR: Is a domain expired is a surprisingly hard question.
